draft-irtf-panrg-path-properties.md

```diff
@@ -129,7 +130,10 @@ For example, a path property based on an approximate calculation may describe th
 When a path-aware network exposes path properties to hosts or other entities,
 these entities may use this information to achieve different goals.
 This section lists several use cases for path properties.
+
 Note that this is not an exhaustive list, as with every new technology and protocol, novel use cases may emerge, and new path properties may become relevant.
+Moreover, for any particular technology, entities may have visibility of and control over different path elements and path properties, and consider them on different levels of abstraction.
+Therefore, a new technology may implement an existing use case, but related to different path elements or on a new level of abstraction.
```

Comment:
I would prefer not to say a **new** level of abstraction, since this seems to indicate that our definition does not yet cover this level of abstraction:
"Moreover, for a particular technology, entities may have visibility of and control over different path elements and path properties, and consider them on different levels of abstraction.
Therefore, a new technology may implement an existing use case related to other path elements or on a different level of abstraction."

Reply:
Agreed! Fixed.